python/src/energuide/embedded/door.py

```diff
@@ -0,0 +1,62 @@
+import typing
+from energuide import bilingual
+from energuide import element
+
+
+class _Door(typing.NamedTuple):
+    label: str
+    door_type: bilingual.Bilingual
+    rsi: float
+    height: float
+    width: float
```

Comment:
For `rsi`, `height`, and `width`, we've run into those enough times that we've factored out code in the same way as `bilingual`. 
There is `energuide.embedded.distance` and `energuide.embedded.insulation` that you should be using. Take a look at `energuide.embedded.ceiling.py` for an example